oasis.py

```diff
@@ -35,8 +35,17 @@ def getTempCredentials():
     
         # Retrieve new credentials from Foxx Service
         print("Requesting new temp credentials.")
+        if (tutorialName is not None):
+             body = {
+            "tutorialName": tutorialName
+             }
+             requestBody = json.dumps(body)
```

Comment:
you can pull out the `requestBody = json.dumps(body)`

Reply:
Updated.